clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -1334,4 +1334,13 @@ public static long getDateTime(String timestamp) throws ParseException, IllegalA
     public static <S> Iterator<S> covariantCast(Iterator<? extends S> iterator) {
         return (Iterator<S>) iterator;
     }
+
+    /**
+     * Checks if a string is null, empty or whitespace only.
+     * @param str a string to be checked
+     * @return true if the string is null, empty or whitespace only; otherwise, return false.
+     */    
+    public static boolean isBlank(String str) {
```

Comment:
How about applying this helper method to code base?

Reply:
> How about applying this helper method to code base?
@chia7712  OK.  I will do it on next PR.